internal/ls/completions.go

```diff
@@ -1622,7 +1635,7 @@ func (l *LanguageService) getCompletionEntriesFromSymbols(
 	for name := range maps.Keys(uniques) {
 		uniqueSet.Add(name)
 	}
-	return uniqueSet, sortedEntries
+	return *uniqueSet, sortedEntries
```

Comment:
Is this just so that the uniqueSet can be optional in some way? Could we just use nil/empty?

Reply:
What do you mean? In general I'm avoiding using `*core.Set` because it crashes if you call `set.Has()`. The dereference is here just because `core.NewSetWithSizeHint` returns a pointer.

Reply:
I see, yeah. I guess we just have a mix of places that need to read or write this so what pointer-ness to use isn't super clear.